OpenShift/helm/nexus-repository-manager/values.yaml

```diff
@@ -7,10 +7,13 @@ deploymentStrategy: {}
   #   maxSurge: 25%
   #   maxUnavailable: 25%
   # type: RollingUpdate
+image:
+  repository: sonatype/nexus3
+  pullPolicy: IfNotPresent
 
 nexus:
-  imageName: registry.connect.redhat.com/sonatype/nexus-repository-manager
-  imageTag: 3.21.1-01-ubi-23
+  imageName: sonatype/nexus3
+  imageTag: 3.21.1
```

Comment:
like in the iq values, it would be good for operators to have imageTag and imageName combined into imageName.

Reply:
I'm not sure about that
https://helm.sh/docs/chart_best_practices/pods/#images